src/main/java/frc/robot/commands/SetGlobalPoseToTagGlobalPose.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.util.coprocessor.networktables.ScorpionTable;
+
+
+/*
+    CHARGED UP 2023 FIELD
```

Comment:
We've agreed on a different coordinate scheme, so this diagram should be updated.

```
    //      CHARGED UP 2023 FIELD
    //    BLUE                   RED
    //  |----------------------------|
    //  |- 5                      4 -|
    //  |              +Y            |
    //  |-- 6          |        3 -- |
    //  |              o--+X         |
    //  |-- 7                   2 -- |
    //  |                            |
    //  |-- 8                   1 -- |
    //  |----------------------------|
    //          SCORING TABLE

```